preview_generator/preview/builder/office__libreoffice.py

```diff
@@ -19,6 +20,8 @@
 from preview_generator.preview.builder.document_generic import write_file_content
 
 
+LOCK = threading.Lock()
```

Comment:
@jumenzel Can you rename this lock to something more explicit. 'libreoffice_call_lock' maybe.

Reply:
Should I add a comment here as well to explain the motivation for the lock? I'm not sure why libreoffice behaves this way. When multiple threads cause multiple overlapping calls some files don't seem to get converted. There's no exit code != 0. So libreoffice isn't complaining about anything.

Reply:
Yes, you can. i do think libreoffice is made somehow that when you start multiple instance, it try to reuse the already started one. i just created an issue about this : https://github.com/algoo/preview-generator/issues/77

So add comments, something like:
```
# INFO - jumenzel - 2019-03-12 - do now allow running 2 libreoffice at the same time to avoid issue.
# INFO - jumenzel - 2019-03-12 - Should we allow running multiple libreoffice instance ? see https://github.com/algoo/preview-generator/issues/77
```